doc/man/cockpit.conf.xml

```diff
@@ -123,6 +123,24 @@ ForwardedForHeader = X-Forwarded-For
             <emphasis>Connect to</emphasis> option to specify the host to log into.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><option>AllowMultiHost</option></term>
+        <listitem>
+          <para>
+            When set to <literal>true</literal>, cockpit will allow
+            users to connect to multiple hosts in one session. The
+            default is OS specific.
+          </para>
+          <para>
+            Connecting to multiple hosts in one session might be
+            problematic since the browser will load JavaScript code
+            from these hosts and execute it all without isolation from
+            each other. This gives one host full access to all other
+            hosts in that session, exposing every host to bugs and
+            malice served from any other host.
```

Comment:
This language needs a little massaging. Perhaps:

> Connecting to untrusted hosts runs JavaScript code from these hosts without isolation from each other. This gives all host full access to all others in that session, including faulty or even malicious actions. Enable this `<emphasis>`only`</emphasis>` if you can put the same level of trust to all remote hosts as the local one.

I.e. avoid some repetition and redundancy (still not perfect), avoid jargon ("bugs"), and give a criterion for when to enable this.

Perhaps @garrett can help here as well.

Reply:
> Connecting to untrusted hosts runs JavaScript...

Well, it also runs it non-isolated when the hosts are trusted... :-)  I think we should describe here what happens on a technical level, and not get too much into trust issues.

Reply:
Yes, describing is good, but this needs some guidance when it's safe to turn on -- and that can *only* be based on trust.

Reply:
Shorter and to the point:

```suggestion
            When connecting to multiple servers, JavaScript from each runs
            without isolation. This increases the risk of software errors 
            and malicious code running on all connected machines.
            Enable this option **only** when you trust all hosts.
```

Even a little shorter, removing the bugs part entirely (as that isn't really the concern here, as all software has bugs anyway):

```suggestion
            When connecting to multiple servers, JavaScript runs without
            isolation, making your system vulnerable to potential attacks
            from one host to another.
            Enable this option **only** when you trust all hosts.
```

I force-wrapped the "Enable" warning to make it stand out more as its own thing, as this is pre-formatted text.

Reply:
Thanks @garrett : This is sort of my "best of" from both proposals:

> When connecting to multiple servers, JavaScript from each runs without isolation. This makes your system and all connected servers vulnerable to attacks from all other hosts. Enable this option **only** when you trust all hosts.

Reply:
OK, how's this?

```suggestion
            When connecting to multiple servers, JavaScript runs without
            isolation. All systems will be vulnerable to potential attacks 
            from other connected hosts. Enable this option **only** when all
            hosts are trusted.
```